airflow/providers/amazon/aws/example_dags/example_emr_job_flow_manual_steps.py

```diff
@@ -50,54 +49,58 @@
         'InstanceGroups': [
             {
                 'Name': 'Primary node',
-                'Market': 'SPOT',
+                'Market': 'ON_DEMAND',
                 'InstanceRole': 'MASTER',
-                'InstanceType': 'm1.medium',
+                'InstanceType': 'm5.xlarge',
```

Comment:
These are the new default/recommended settings.  The m1 series instance types are "previous generation" and no longer recommend.